### ISSUE TITLE
Can we add an option to just display the important R commands?

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#8177]
@ChrisMarsh82 and @lloyddewit I don't even know if this is possible?

I have now switched off the display of commands in the results window.   I then didi a calculation, i.e. 2 + 3.  In the output window I had the following:

![image](https://user-images.githubusercontent.com/11226469/221977957-26f8c5d6-0639-4c64-b8bd-4d329a74ccd5.png)

So I switched on the commands so I could see the expression.  Here it is:

![image](https://user-images.githubusercontent.com/11226469/221978405-95914c16-f336-4a31-ba73-1cb99e048c60.png)

It is nice that I have the 2 + 3 + 4, so I know what the expression is, but I always also have all the housekeeping commands as well.

This is just about the same for each dialogue, i.e. we a) get the data from a data frame - special R-Instat function.
b) Do something
c) Tidy up with more special R-Instat functions.

Could we have a halfway option on displaying the commands to just display b).

We will always have the whole thing in the script and log windows.  But that would be more attractive in the results windows.

**Comment by lloyddewit** [comment]
@rdstern I don't think this issue requires any changes to the Log/script window. So I removed the `log_script window` label. I hope that's OK. If I misunderstood, then please restore the label, thanks.
